tests/test_threadpoolctl.py

```diff
@@ -263,6 +263,8 @@ def test_multiple_shipped_openblas():
 
 
 @pytest.mark.skipif(scipy is None, reason="requires scipy")
+@pytest.mark.skipif(not cython_extensions_compiled,
+                    reason='Requires cython extensions to be compiled')
```

Comment:
Why is that necessary now ?

Reply:
If you do not build the extension locally, no?